hypothesis-python/tests/cover/test_conjecture_engine.py

```diff
@@ -766,6 +768,26 @@ def f(data):
     assert not runner.prescreen_buffer(hbytes([255] * 7))
 
 
+@given(st.integers(0, 255), st.integers(0, 255))
+def test_prescreen_with_capped_byte_agrees_with_results(byte_a, byte_b):
```

Comment:
This is a nice test!

I think it should probably go in the `nocover` version of this file (i.e. `tests/nocover/test_conjecture_engine.py`. We're trying to get the runtime of `tests/cover` down, and I don't think this is needed to actually trigger coverage of anything, while the use of `@given` makes it vaguely expensive.

Reply:
In that case, I need to ask how to actually run the `nocover` tests locally.

So far I've just been using `./build.sh check-coverage` and `check-py36`, and the latter doesn't seem to include the `nocover` suite.

(I put a sabotaged test in `nocover/test_conjecture_engine.py`, and all of my test runs still pass.)

I'm on Mac OS, if that's relevant.

Reply:
I think `./build.sh py36-full` should do the trick?

Reply:
That doesn't seem to be a valid task for `build.sh`.

At the end of my local run of `check-py36`, I see `py36-full: commands succeeded`, but there's no sign of my sabotaged test having failed at all.

Reply:
https://github.com/HypothesisWorks/hypothesis/pull/1498

(Thanks for the virtualenv pointer; I now have that working too.)